modules/rubiconBidAdapter.js

```diff
@@ -1305,4 +1186,70 @@ export function resetUserSync() {
   hasSynced = false;
 }
 
+function setBidFloors(bidRequest, imp) {
+  if (imp.bidfloorcur != 'USD') {
+    delete imp.bidfloor;
+    delete imp.bidfloorcur;
+  }
+
+  let bidFloor = parseFloat(deepAccess(bidRequest, 'params.floor'));
+
+  if (!isNaN(bidFloor)) {
+    imp.bidfloor = bidFloor;
+    imp.bidfloorcur = 'USD';
+  }
```

Comment:
`setBidFloors` seems like it will overwrite the price_floors module if bidder params floor is present.

I think we want to keep the price_floors module's floors if it is present and USD. and if NOT, then we should check our param floor.

Reply:
It looks like I reverted the original if/else. So, the behaviour should be: if there's a floor specified through the module, just use it. Otherwise, try to read the params.floor (which bret wants to deprecate for pbjs 8.0) . Let me adjust this.

Reply:
👍